server/models/games/sueca.js

```diff
@@ -25,7 +25,11 @@ class Sueca extends Game {
     for (let i = 0; i < 4; i++) {
       this.hands[i] = deck.slice(10 * i, 10 * (i + 1));
     }
-    this.trumpPlayer = Math.floor(Math.random() * 4 + 1);
+    if (params.lastGame && params.lastGame.trumpPlayer) {
+      this.trumpPlayer = ( params.lastGame.trumpPlayer % 4 ) + 1;
```

Comment:
I believe you can use Sueca.getPlayerAfter(params.lastGame.trumpPlayer) here instead of calculating it manually, can't you?

Also, didn't eslint complain about putting spaces inside parenthesis? It it didn't it should have :P